.github/workflows/continuous-integration-workflow.yaml

```diff
@@ -96,6 +96,21 @@ jobs:
           command: test
           args: --no-default-features
 
+  kani:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Verify with Kani
+        uses: model-checking/kani-github-action@02764e122079011adaf31e7d0630a8841d277507
+        with:
+          args: |
+            --tests -p prost-types --default-unwind 3 \
+            --harness "tests::check_timestamp_roundtrip_via_system_time"
```

Comment:
What will happen if more than one test needs to be run?

Reply:
With the current `cargo kani` arguments, you would need to repeat lines 106-111 if you want to run exactly one more test case. Running all tests in `prost-types` can be done by just removing line 111.

I think may be good to adjust the definition of `--harness` to allow for multiple such arguments to be passed. [Currently at most one `--harness` can be used.](https://github.com/model-checking/kani/blob/967e41d8e35865b8360ff54ec65128bc84eb74aa/kani-driver/src/args.rs#L126)

Reply:
Tracking with #2118